app/controllers/admin/instances/moderation_notes_controller.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+
+class Admin::Instances::ModerationNotesController < Admin::BaseController
+  before_action :set_instance, only: [:create]
+  before_action :set_instance_note, only: [:destroy]
+  rescue_from ActiveRecord::RecordInvalid, with: :bad_request
```

Comment:
Can you remove this line?

Reply:
Oops, didn't even remember it was there.

Reply:
Have removed from both the controllers.